src/bin/flamegraph.rs

```diff
@@ -131,6 +133,18 @@ struct Opt {
     )]
     fontwidth: f64,
 
+    /// Color of UI text such as the search and reset zoom buttons
+    #[clap(
+        long = "uicolor",
+        default_value = defaults::UI_COLOR,
+        value_parser = |s: &str| {
+            parse_hex_color(s)
+                .ok_or_else(|| format!("unknown ui color: {}", s))
```

Comment:
Let's be a little more helpful here. Something along the lines of "expected a color in hexadecimal format"